src/size-rhythms.md

```diff
@@ -0,0 +1,65 @@
+# Size rhythms
+
+When designing components and layouts, you will have to specify size of many properties such as `padding`, `margin`, `grid-gap`, `width`, `height`, and so forth.
+
+To generate a consistent layout and visual expression, we only use sizes that are multiples of four (4 * x). For example 4, 8, 12, 16, 20, and so on.
+
+It is very important to follow this convention, both when creating new components for Lime Elements, and when creating bigger components and layouts which use these components. When the entire interface is calculated based on the single core value of 4, elements of the UI will align auto-magically.
+
+For instance, if you want to create a custom row of buttons (not directly using Lime Elements) with this sizing convention, its `height` and `line-height` can be 40 units (to vertically align the text inside the button), the `padding-left` and `padding-right` can be 12 units, and its `margin-left` and `margin-right` which set the distance to the next or previous buttons can be for example 4 or 2, or in this example 6.
+
+Even though 6 is not a multiple of 4, it sums up to a 12 units distance between two buttons in a row; and since 12 is a multiple of 4, the result will be moa👌.
+
+<limel-example-size></limel-example-size>
+
+
+## REM or PX
+We only use `rem` units in our designs system, based on browsers' default base unit of measurement (where `16px` = `1rem`). In such a system, multiples of 4px (our core size unit) can easily be calculated, as they increase sequentially by `0.25`.
+
+* 24 px = 1.5 rem
+* 20 px = 1.25 rem
+* 16 px = 1 rem
+* 8 px = 0.5 rem
+* 4 px = 0.25 rem
+
+However, while designing a UI, it may be easier to think in pixels, as rem is more abstract. By using a custom SASS function like below, you can effortlessly leave the job of unit conversion to the computer.
+
+````css
+/// first define this converter function in your SCSS ···
+@function pxToRem($px) {
+    @return #{$px/16}rem;
+}
+
+// ··· and then use it to ease calculations
+.example {
+    width: pxToRem(4); // will be compiled to 0.25rem
+}
+````
+
+If you are developing new components for Lime Elements, you should know that this function is already available and can be easily imported into your `.scss` files via: `@import '../../style/mixins';`
+
+So if you think something should be rendered 40px wide on the screen, you you don't need to calculate the respective value in REM. Just type `width: pxToRem(40);` in your SCSS code.
```

Comment:
I'm not sure we should include documentation only directed at developers of lime-elements itself here. We could perhaps include it on the *Contributing* page…

Reply:
Good idea! Will you update the contribution page?

Reply:
I'll take a look at it.

Reply:
I'll add this:

> ### Styling
>
> Please read the Design Guidelines section of the documentation, look at existing code, and feel free to ask any questions on the issue or PR you are working on.

Any contributor who is thorough enough to read that file, is likely to look at existing code and find the pxToRem function. And if they don't, we'll just point it out in the PR review.

Reply:
(In other words, I still think we should remove the above two paragraphs.

```suggestion
```

Reply:
I pushed a fixup removing this 👍